### PR TITLE
[4.0] com_users multiple groups

### PR DESCRIPTION
**Pull request by brianteeman**
In the list of users if a user is in more than 2 groups then it just displays "Multiple groups" and then a tooltip listing the groups.

This PR changes it from a completely inaccessible tooltip to the new div approach and instead of putting the groups on new lines it correctly puts them in a list. Also adds tabindex so that keyboard users can access this information.

### After
image

### CODE REVIEW

File: administrator/components/com_users/tmpl/users/default.php [hunk 1]
```diff
@@ -172,7 +172,11 @@
 								<?php endif; ?>
 								<td>
 									<?php if (substr_count($item->group_names, "\n") > 1) : ?>
-										<span class="hasTooltip" title="<?php echo HTMLHelper::_('tooltipText', Text::_('COM_USERS_HEADING_GROUPS'), nl2br($item->group_names), 0); ?>"><?php echo Text::_('COM_USERS_USERS_MULTIPLE_GROUPS'); ?></span>
+										<span tabindex="0"><?php echo Text::_('COM_USERS_USERS_MULTIPLE_GROUPS'); ?></span>
+										<div role="tooltip" id="tip<?php echo $i; ?>">
+											<strong><?php echo Text::_('COM_USERS_HEADING_GROUPS'); ?></strong>
+											<ul><li><?php echo str_replace("\n",'</li><li>', $item->group_names); ?></li></ul>
```
Comment (Quy) [inline administrator/components/com_users/tmpl/users/default.php]:
```suggestion
											<ul><li><?php echo str_replace("\n", '</li><li>', $item->group_names); ?></li></ul>
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on df46f05f452de1707684b3b6d4513f218275c2db<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25230">issues.joomla.org/tracker/joomla-cms/25230</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**